deploy/olm-catalog/ibm-block-csi-operator-community/1.3.0/csi.ibm.com_ibmblockcsis_crd.yaml

```diff
@@ -2,6 +2,13 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: ibmblockcsis.csi.ibm.com
+  labels:
+    app.kubernetes.io/name: ibm-block-csi-operator
+    app.kubernetes.io/instance: ibm-block-csi-operator
+    app.kubernetes.io/managed-by: ibm-block-csi-operator
+    product: ibm-block-csi-driver
+    release: v1.3.0
+    csi: ibm
```

Comment:
could you do the same for 1.4.0?

Reply:
can't find the file csi.ibm.com_ibmblockcsis_crd.yaml under 1.4.0 folder

Reply:
well, it's [definitely there](https://github.com/IBM/ibm-block-csi-operator/blob/f95799fb0cff3cc2246fa9d4bd1ca4b371b973bc/deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/csi.ibm.com_ibmblockcsis_crd.yaml)

Reply:
no, just update under 1.4.0

Reply:
done